### ISSUE TITLE
cache sessions in DB

### ISSUE THREAD

**Issue by bebehei** [kristophjunge/docker-mediawiki#4]
> Known issues
> - Sessions are stored in memory via APC and will be lost after container stop.

What about using [`$wgSessionCacheType = CACHE_DB;`](https://www.mediawiki.org/wiki/Manual:$wgSessionCacheType) in `LocalSettings.php`? According to the docs, this should save session tokens in Database and database should be persistent and survive a reboot.

I tested it with `ExtraLocalSettings.php` and it seems to work. I couldn't verify it by 100% (it's 3am).

FB to your image: Keep up the good work! This docker image is absolutely high quality! 👍

**Comment by kristophjunge** [comment]
I have tested it and can confirm its working. 

I decided to make it a MySQL specific default value for the following reasons:
- The [documentation](https://www.mediawiki.org/wiki/Manual:$wgSessionCacheType) says that it must be set to a persistent cache for proper operation.
- I think its the expected default behaviour when using MySQL that the sessions are persistent and even when its not the most performant cache its at least scaleable.

I decide not to use it as default when using SQLite because it would lead to a large amount of write operations which could make the usage of SQLite inefficient.

I removed the "Known Issue" from the readme and added you to the contributors list.

Thank you very much!
